pages/events.js

```diff
@@ -20,6 +21,18 @@ export async function getServerSideProps(context) {
 }
 
 export default function Events({ events }) {
+  const [eventType, seteventType] = useState("all");
+  let categorisedEvents = {
+    all: events,
+    virtual: events.filter((event) => event.isVirtual === true),
+    inPerson: events.filter((event) => event.isInPerson === true),
+    cfpOpen: events.filter(
+      (event) => new Date(event.date.cfpClose) > new Date()
```

Comment:
Is `date.cfpClose` a required field?

Reply:
Good spot, it is not required, so I should make a check 👍